app/server/services/auth.ts

```
@@ -0,0 +1,133 @@
+  data: formData,
+}: {
+  data: SignUpFormType;
+}): Promise<AuthResponse> => {
+  const isEmailInput = isEmail(formData.emailOrUsername);
+
+  const provider = isEmailInput ? "email" : "username";
+
+  const placeholderEmail = `${formData.emailOrUsername}@gymworld.com`;
+
+  const formToDatabase = formDataToDatabase(formData);
+  delete formToDatabase.emailOrUsername;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)